### PR TITLE
docs: fix readme grammar typos

### PR DESCRIPTION
**Pull request by prateek951**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -29,9 +29,9 @@ Renovate was released in 2017 and already is widely used in the developer commun
 ## The Renovate Approach
 
 - We believe everyone can benefit from automation, whether it's a little or a lot
-- Renovate should not cause you to change your workflow against your wishes, instead it should be adaptable to your existing workflow
-- All behaviour should be configurable, down to a ridiculous level if necessary
-- Autodetect settings wherever possible (to minimise configuration) but always allow overrides
+- Renovate should not cause you to change your workflow against your wishes instead, it should be adaptable to your existing workflow
```
Comment (rarkins) [inline readme.md]: No. revert

File: readme.md [hunk 2]
```diff
@@ -29,9 +29,9 @@ Renovate was released in 2017 and already is widely used in the developer commun
...
+- All behavior should be configurable, down to a ridiculous level if necessary
+- Autodetect settings wherever possible (to minimize configuration) but always allow overrides
```
Comment (rarkins) [inline readme.md]: Please don't change documentation based on american vs british spelling.

**Review by rarkins — CHANGES_REQUESTED** [review]
Revert undesired changes

File: readme.md (hunk 2 above)
Comment (prateek951) [reply readme.md]: Noted @rarkins

**Pull request closed without merging**